src/fulltext.js

```diff
@@ -1,4 +1,4 @@
-var _ = require('./../lib/lodash');
+var _ = require('./../vendor/lodash');
```

Comment:
Moved this to `vendor` instead of `lib` since lib will include the transpiled assets for npm packages via the `package.json#main` property.